### PR TITLE
fix(lint): update linting in metrics-test

### PR DESCRIPTION
**Pull request by jayjayjpg**
## Description of change

This fixes up a small linter error on the CLI.

```bash
$ golangci-lint run

 cmd/meroxa/global/metrics_test.go:96:18: dynamicFmtString: use errors.New(err) or fmt.Errorf('%s', err) instead (gocritic)
addError(event, fmt.Errorf(err))
```

## Type of change

- [ ]  New feature
- [ ]  Bug fix
- [ ]  Refactor
- [ ]  Documentation
- [x]  Chore

## How was this tested?

- [x]  Run linter locally
- [ ]  Tested in staging

### CODE REVIEW

File: cmd/meroxa/global/metrics_test.go [hunk 1]
```diff
@@ -93,7 +93,7 @@ func TestAddError(t *testing.T) {
 	}
 
 	err := "unexpected error"
-	addError(event, fmt.Errorf(err))
+	addError(event, fmt.Errorf("%s", err))
```
Comment (dianadoherty) [inline cmd/meroxa/global/metrics_test.go:96]: Wouldn't this do the trick?
```suggestion
	addError(event, err)
```

Comment (jayjayjpg) [reply cmd/meroxa/global/metrics_test.go:96]: Yes, this reads great -- thank you!

Comment (jayjayjpg) [reply cmd/meroxa/global/metrics_test.go]: Woops, it seems that will fail the tests:

```
  Error: cannot use err (variable of type string) as error value in argument to addError: missing method Error (typecheck)
  
  Error: issues found
```

Should we use the `addError(event, fmt.Errorf("%s", err))` syntax instead? What do you think?

File: cmd/meroxa/global/metrics_test.go [hunk 2]
```diff
@@ -93,7 +93,7 @@ func TestAddError(t *testing.T) {
 	}
 
 	err := "unexpected error"
-	addError(event, fmt.Errorf("%s", err))
+	addError(event, err)
```
Comment (dianadoherty) [inline cmd/meroxa/global/metrics_test.go:96]: oh wait sorry I was blind:
```
err := errors.New("unexpected error""
addError(event, err)
```

Comment (jayjayjpg) [reply cmd/meroxa/global/metrics_test.go]: Got it, thank you!

Comment (jayjayjpg) [reply cmd/meroxa/global/metrics_test.go]: Perfect, this works great and thank you for the outline!

**Review by dianadoherty — APPROVED** [review]

**Pull request merged**